common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -0,0 +1,350 @@
+        self.assertTrue(self.my_profile_page.mode_for_field('language_proficiencies'), 'edit')
+
+        self.assertTrue(self.my_profile_page.field_icon_present('language_proficiencies'))
+
+    def test_about_me_field(self):
+        """
+        Test behaviour of `About Me` field.
+
+        Given that I am a registered user.
+        And I visit My Profile page.
+        And I set the profile visibility to public and set default values for public fields.
+        Then I set about me value to `Eat Sleep Code`.
+        Then displayed about me should be `Eat Sleep Code` and about me field mode should be `display`
+        And I reload the page.
+        Then displayed about me should be `Eat Sleep Code` and about me field mode should be `display`
+        Then I set empty value for about me.
+        Then displayed about me should be `Tell other edX learners a little about yourself: where you live,
+        what your interests are, why you're taking courses on edX, or what you hope to learn.` and about me
+        field mode should be `placeholder`
+        And I reload the page.
+        Then displayed about me should be `Tell other edX learners a little about yourself: where you live,
+        what your interests are, why you're taking courses on edX, or what you hope to learn.` and about me
+        field mode should be `placeholder`
+        And I make `about me` field editable
+        Then `about me` field mode should be `edit`
+        """
+        placeholder_value = (
+            "Tell other edX learners a little about yourself: where you live, what your interests are, "
+            "why you're taking courses on edX, or what you hope to learn."
+        )
+
+        self._test_textarea_field('bio', 'Eat Sleep Code', 'Eat Sleep Code', 'display')
+        self._test_textarea_field('bio', '', placeholder_value, 'placeholder')
+
+        self.my_profile_page.make_field_editable('bio')
+        self.assertTrue(self.my_profile_page.mode_for_field('bio'), 'edit')
+
+    def test_birth_year_not_set(self):
+        """
+        Verify message if birth year is not set.
+
+        Given that I am a registered user.
+        And birth year is not set for the user.
+        And i visit my profile page.
+        Then i should see message `Your profile is disabled because you haven't filled in your Year of Birth.`
+        """
+        message = "You must specify your birth year before you can share your full profile."
+        self.verify_profile_forced_private_message('', message=message)
+
+    def test_user_is_under_age(self):
+        """
+        Verify message if user is under age.
+
+        Given that I am a registered user.
+        And birth year is set so that age is less than 13.
+        And i visit my profile page.
+        Then i should see message `You must be over 13 to share a full profile.`
+        """
+        self.verify_profile_forced_private_message('2010', message='You must be over 13 to share a full profile.')
```

Comment:
I think you need to change this so that 2010 is not hard-coded. This test will start failing in 2024. And yes, I'm sure this code will still be running then. :)